MaraBot/Commands/CompletedCommandModule.cs

```diff
@@ -45,14 +45,20 @@ public class CompletedCommandModule : BaseCommandModule
             Permissions.AccessChannels)]
         public async Task Execute(CommandContext ctx, TimeSpan time)
         {
+            // Give a sufficient delay before deleting message
+            await Task.Delay(500);
```

Comment:
A 500ms delay might be a lot depending on if Discord sends out push notifications early enough. Let's keep it at this for now, but if I start seeing push notifications for these, I'll make the delay a bit shorter.
I can easily test it, because I have push notifications on server wide :).